lib/appium.js

```diff
@@ -173,6 +173,10 @@ class AppiumDriver extends BaseDriver {
     if (!_.isString(caps.platformName)) {
       throw new Error('You must include a platformName capability');
     }
+    if (_.isUndefined(caps.automationName)) {
```

Comment:
I'd rather add the warning below in case `return` condition didn't match anything. We also would like to say something about incorrect values there

Reply:
I agree. This makes it seem like there is an alternative for some platforms, like if I wanted iOS 7 using Instruments I could supply _some_ `automationName` to get it, but that is not correct.

I would be happy to have an `automationName` for every platform, though, and then prefer it.

Reply:
That's a good point. I'll give legacy IOS and Android drivers automation names....

What do you think about:
android = uiautomator1
ios=uiautomation

Reply:
👍 i'm ambivalent between `UIAutomation` and `Instruments` for iOS though.

Reply:
`instruments` is probably better.... distinguishes it from Android